src/jesse.app.src

```diff
@@ -1,10 +1,14 @@
 %%-*- mode: erlang -*-
 
 { application, jesse, [ {description, "jesse"}
-                      , {vsn, git}
+                      , {vsn, "0.1.3"}
```

Comment:
i was considering making this change for other reasons (using jesse in a non-git env), but once again spread some knowledge. sorry if it seems like nitpicking, I'm not. Thanks!